test/GoTrueClient.test.ts

```diff
@@ -221,19 +238,21 @@ describe('GoTrueClient', () => {
     })
   })
 
-  test('signUp() with email', async () => {
-    const { email, password } = mockUserCredentials()
+  describe('Email Auth', () => {
+    test('signUp() with email', async () => {
+      const { email, password } = mockUserCredentials()
 
-    const { error, data } = await auth.signUp({
-      email,
-      password,
-    })
+      const { error, data } = await auth.signUp({
+        email,
+        password,
+      })
 
-    expect(error).toBeNull()
-    expect(data.session).not.toBeNull()
-    expect(data.user).not.toBeNull()
+      expect(error).toBeNull()
```

Comment:
is the extra space intended?

Reply:
extra space here is due to it being wrapped by line 242